src/jdk.incubator.concurrent/share/classes/jdk/incubator/concurrent/StructuredTaskScope.java

```diff
@@ -797,6 +798,19 @@ public String toString() {
         }
     }
 
+    private static int stateToId(Future.State s) {
```

Comment:
Did you mean to to name this stateToId or stateToInt? 
Style-wise I guess I'd would align the arrows but it doesn't matter.